l5x/test.L5X

```diff
@@ -14,6 +14,48 @@
 <Member Name="DayOW1" DataType="REAL" Dimension="0" Radix="Float" Hidden="false" ExternalAccess="Read/Write"/>
 </Members>
 </DataType>
+<DataType Name="datas_for_eating" Family="NoFamily" Class="User">
+<Members>
+<Member Name="XprivateX_cleaning_c0" DataType="SINT" Dimension="0" Radix="Decimal" Hidden="true" ExternalAccess="Read/Write"/>
+<Member Name="DEMAND" DataType="BIT" Dimension="0" Radix="Decimal" Hidden="false" Target="XprivateX_cleaning_c0" BitNumber="0" ExternalAccess="Read/Write">
+<Description>
+<![CDATA[How much eating is demanded?]]>
+</Description>
+</Member>
+<Member Name="FOOD_TIMER" DataType="TIMER" Dimension="0" Radix="NullType" Hidden="false" ExternalAccess="Read/Write">
+<Description>
+<![CDATA[Timer tracking the food consumption]]>
+</Description>
+</Member>
+<Member Name="MEAL_PREP_TIMER" DataType="TIMER" Dimension="0" Radix="NullType" Hidden="false" ExternalAccess="Read/Write">
+<Description>
+<![CDATA[Timer for amount of meal prep time]]>
+</Description>
+</Member>
+<Member Name="BHAIG29GI" DataType="TIMER" Dimension="0" Radix="NullType" Hidden="false" ExternalAccess="Read/Write">
```

Comment:
Another radix question: does this mean that, effectively, the value is not a numeric one (ie. it has a null radix)?  If so, can this be elided (or otherwise made a default value) somehow, perhaps?

Reply:
I believe `NullType` indicates the data type is an object/struct. (Yes, they probably should have called it `ObjectType`.) So I think including that information is important. (But my goal was just to parse the data, not to necessarily understand it.)

Reply:
Yep, makes sense.  If this is the raw XML coming from the plant, I agree then that the right thing to do is not think too hard about it :-)